lemarche/templates/tenders/list.html

```diff
@@ -35,20 +35,73 @@ <h1>{{ page_title }}</h1>
                         <i class="ri-add-fill ri-lg mr-2"></i>Publier un besoin d'achat
                     </a>
                 </div>
-            {% endif %}
-            <div class="col-12 my-5 py-3">
-                <!-- "buyer": display tenders which the user is the author -->
-                {% if user.kind != user.KIND_SIAE %}
-                    {% for tender in tenders %}
-                        {% include "tenders/_list_item_buyer.html" with tender=tender %}
-                    {% endfor %}
-                    {% include "includes/_pagination.html" %}
-                    {% if not tenders %}
-                        <p class="text-muted">
-                            Publiez un besoin pour trouver des prestataires inclusifs près de chez vous.
-                        </p>
-                    {% endif %}
+                {% endif %}
+                <div class="col-12 my-5 py-3">
+                    <!-- "buyer": display tenders which the user is the author -->
+                    {% if user.kind != user.KIND_SIAE %}
+                    {% block htmx %}
+                    <div id="tendersList">
+                        <ul role="navigation" class="nav nav-tabs nav-tabs--marche">
+                            {% url 'tenders:list' as TENDERS_LIST_URL %}
+                            {% url 'tenders:list' status=STATUS_DRAFT as TENDERS_DRAFT_LIST_URL %}
+                            {% url 'tenders:list' status=STATUS_PUBLISHED as TENDERS_PUBLISHED_LIST_URL %}
+                            {% url 'tenders:list' status=STATUS_VALIDATED as TENDERS_VALIDATED_LIST_URL %}
+
+                            <li class="nav-item">
+                                <a role="button" hx-push-url="true" hx-get="{{TENDERS_LIST_URL}}"
+                                    class="nav-link{% if request.get_full_path == TENDERS_LIST_URL %} active{% endif %}"
+                                    hx-target="#tendersList" hx-swap="outerHTML">
+                                    Tous
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a role="button" hx-push-url="true" hx-get="{{TENDERS_DRAFT_LIST_URL}}"
+                                    class="nav-link{% if request.get_full_path == TENDERS_DRAFT_LIST_URL %} active{% endif %}"
+                                    hx-target="#tendersList" hx-swap="outerHTML">
+                                    Brouillons
```

Comment:
```suggestion
                                    Brouillon
```
vu que les autres sont au singulier